Bazarr: Initial release (0.9.9)

smaarn opened the pull request:
_Motivation:_  Creating initial bazarr spk
_Linked issues:_  Fixes #3535. Supersedes #4629 , #4453, #4547, #4751

Includes:

* New version of `gevent` (21.x)
* New version of `greenlet` (1.x)
* New version of `libev` (4.33 as opposed to current 4.22 used by `ffsync`)
* `zope.event`: 4.5.0
* `gevent-websocket`: 0.10.1

### Checklist
- [x] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [ ] New installation of package completed successfully

th0ma7 commented:
@smaarn I suggest you have a look at PR #4695 merged previously.  I've fixed `ffsync` and updated `gevent`, `greenlet` and `libev` to level you'll find acceptable (or similar or more recent).

smaarn commented:
> @smaarn I suggest you have a look at PR #4695 merged previously. I've fixed `ffsync` and updated `gevent`, `greenlet` and `libev` to level you'll find acceptable (or similar or more recent).

Thanks for the hint ! 
Sorry I hadn't spotted it.

smaarn commented:
Superseded by #4916
